source/data-api/data-formats.txt

```diff
@@ -0,0 +1,369 @@
+.. _data-api-data-formats:
+
+============
+Data Formats
+============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+MongoDB stores data in a format called BSON, which is similar to a JSON
+object in structure but supports additional data types and uses a binary
+encoding. BSON is efficient for computers but is not human readable so
+you can't work with it directly.
+
+Instead, the Data API uses two formats to represent data in requests and
+responses:
+
+- The **JSON** format uses standard types that any tool can parse and
+  understand. However, JSON cannot represent all BSON types so JSON
+  responses may lose type information for some fields. For example, BSON
+  has distinct types for 32-bit integers and 64-bit floats but a JSON
+  response represents both as a ``number``.
+
+- The **EJSON** format, short for :manual:`MongoDB Extended JSON
+  </reference/mongodb-extended-json/>`, is a superset of standard JSON
+  that uses structured fields to represent BSON data that don't have
+  corresponding JSON types. This fully represents your data but requires
+  your client to understand how to work with EJSON.
```

Comment:
doesn't need to be in this PR, but would be helpful to point users to docs on how to work w EJSON from client code.

Reply:
Don't think we actually have a great article on this. Not sure if we want it in this section or elsewhere but would be great to have.